tahoe_idp/magiclink_helpers.py

```diff
@@ -25,9 +24,6 @@ def create_magiclink(
     # Only the last magic link is usable per user
     MagicLink.objects.filter(username=username, used=False).update(used=True)
 
-    if not redirect_url:
-        redirect_url = get_url_path(settings.LOGIN_REDIRECT_URL)
```

Comment:
the buggy line

Reply:
I fixed this bug in my dream last night @OmarIthawi , thank you for making that a reality 😁